Create announcements 01-25-2021

jeremywalter opened the pull request.

josesanchez1815 reviewed (commented):
I think it may sound better to say "The CFDE Cross-Pollination Series continues next week on Tuesday, February 2, at 11am(PST)/2pm(EST) and LINCS, SPARC, and NIH representatives are scheduled to present."

"Cross-Pollination events take place on the first Tuesday of every month. LINCS, SPARC, and the NIH are our scheduled presenters." sounds kind of weird and implies that they are leading all the presentations, I feel.

jeremywalter commented:
good catch, I had moved things badly... changed it to be a little better (I
hope)

On Tue, Jan 26, 2021 at 2:33 PM Jose Manuel Sanchez, Jr. <
notifications@github.com> wrote:

> *@josesanchez1815* commented on this pull request.
>
> I think it may sound better to say "The CFDE Cross-Pollination Series
> continues next week on Tuesday, February 2, at 11am(PST)/2pm(EST) and
> LINCS, SPARC, and NIH representatives are scheduled to present."
>
> "Cross-Pollination events take place on the first Tuesday of every month.
> LINCS, SPARC, and the NIH are our scheduled presenters." sounds kind of
> weird and implies that they are leading all the presentations, I feel.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nih-cfde/announcements/pull/78#pullrequestreview-576807289>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJV3OY2H2L3V77FZHFRWGELS3463NANCNFSM4WSUH7DA>
> .
>
